sos/report/plugins/ceph_mgr.py

```diff
@@ -103,17 +72,66 @@ def setup(self):
             "version"
         ]
 
+        directory = ''
+        if not microceph_pkg:
+            directory = '/var/run/ceph'
+            self.add_file_tags({
+                '/var/log/ceph/(.*/)?ceph-mgr.*.log': 'ceph_mgr_log',
+            })
+
+            self.add_forbidden_path([
+                "/etc/ceph/*keyring*",
+                "/var/lib/ceph/**/*keyring*",
+                "/var/lib/ceph/**/osd*",
+                "/var/lib/ceph/**/mon*",
+                # Excludes temporary ceph-osd mount location like
+                # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
+                "/var/lib/ceph/**/tmp/*mnt*",
+                "/etc/ceph/*bindpass*",
+            ])
+
+            self.add_copy_spec([
+                "/var/log/ceph/**/ceph-mgr*.log",
+                "/var/lib/ceph/**/mgr*",
+                "/var/lib/ceph/**/bootstrap-mgr/",
+                "/run/ceph/**/ceph-mgr*",
+            ])
+
+        else:
+            directory = '/var/snap/microceph'
+            self.add_file_tags({
+                '/var/log/ceph/common/logs/ceph-mgr.*.log': 'ceph_mgr_log',
```

Comment:
would the same regex that is above work here? hence could be a common one?

Reply:
can do but imho results in 2 function calls instead of one (1 for specific and 1 for common) so leaving this as it is